app/sagas/createChannel.js

```diff
@@ -110,7 +110,9 @@ const handleSuccess = function* handleSuccess({ data }) {
 
 const handleFailure = function handleFailure({ err, isTeam }) {
 	setTimeout(() => {
-		const msg = err.data.errorType ? I18n.t(err.data.errorType, { room_name: err.data.details.channel_name }) : err.reason || I18n.t('There_was_an_error_while_action', { action: isTeam ? I18n.t('creating_team') : I18n.t('creating_channel') });
+		const msg = (err?.data?.errorType && err?.data?.details?.channel_name)
+			? I18n.t(err?.data?.errorType, { room_name: err?.data?.details?.channel_name })
```

Comment:
We should check the error returned before trying to parse with `I18n.t`.
If the server adds a new error message and the app is outdated, the user is going to see `[missing "en.NEW-ERROR-CODE" translation]`.